docs/topics/multiplatform-mobile/multiplatform-mobile-ktor-sqldelight.md

```diff
@@ -0,0 +1,1069 @@
+[//]: # (title: Create a multiplatfotm app using Ktor and SQLDelight – tutorial)
+
+This tutorial demonstrates how to use Android Studio to create a mobile application for iOS and Android using Kotlin
+Multiplatform Mobile with Ktor and SQLDelight.
+
+The application will include a module with shared code for both iOS and Android platforms. Business logic and data
+access layers are implemented only once in the shared module, while the UI of both applications will be native.
+
+The output will be an app that retrieves data over the internet from a
+public [SpaceX API](https://docs.spacexdata.com/?version=latest), saves it in a local database, and displays a list of
+SpaceX rocket launches together with the launch date, results, and a detailed description of the launch:
+
+![Emulator and Simulator](android-and-ios.png){width=600}
+
+You will use the following multiplatform libraries in the project:
+
+* [Ktor](https://ktor.io/docs/create-client.html) as an HTTP client for retrieving data over the internet.
+* [`kotlinx.serialization`](https://github.com/Kotlin/kotlinx.serialization) to deserialize JSON responses into objects
+  of entity classes
+* [`kotlinx.coroutines`](https://github.com/Kotlin/kotlinx.coroutines) to write asynchronous code
+* [SQLDelight](https://github.com/cashapp/sqldelight) to generate the Kotlin code from SQL queries to create a type-safe
+  database API
+
+> You can find the [template project](https://github.com/kotlin-hands-on/kmm-networking-and-data-storage) as well as the
+> source code of the [final application](https://github.com/kotlin-hands-on/kmm-networking-and-data-storage/tree/final) on
+> the corresponding GitHub repository.
+>
+{type="note"}
+
+## Before you start
+
+1. Download and install [Android Studio](https://developer.android.com/studio/).
+2. Search for
+   the [Kotlin Multiplatform Mobile plugin](https://plugins.jetbrains.com/plugin/14936-kotlin-multiplatform-mobile)
+   in the Android Studio Marketplace and install it for working with the shared module and Android project.
```

Comment:
Maybe a screenshot from the AS Marketplace would be good. Otherwise, users might start looking for an "install" button on the page from the link.